pymc/distributions/continuous.py

```diff
@@ -2532,7 +2532,22 @@ def logcdf(value, alpha, beta):
         )
 
 
-class ChiSquared:
+class ChiSquaredRV(RandomVariable):
```

Comment:
We shouldn't add more Ops just for naming. Less Ops means we can sample in other backends / do rewrites with fewer building blocks

Reply:
Nope, that's just a temporary hack. As soon as you do something like `model.copy()`, or any other model transform it's going to be lost. Why do you need it to show up as ChiSquared in the graphviz? 

It's just a parametrization. Similarly we don't write UnitNormal if it happens to have loc=0, scale=1, even if we offered a helper to create them?

Reply:
I don't think naming should come at the cost of more code complexity/redundancy. This is just a helper that creates a specific Gamma and seems sensible to write it as such. Can't you call pz.plot on the gamma object that's returned?

PS: We can emphasize more in the docstrings that this is just a helper that return a regular Gamma distribution